schemas/fingerprint-server-api.yaml

```diff
@@ -643,6 +643,28 @@ components:
         - visits
       title: PaginatedResponse
       description: Fields `lastTimestamp` and `paginationKey` added when `limit` or `before` parameter provided and there is more data to show
+    ErrorCommon400Response:
```

Comment:
Maybe this should be called `ErrorVisits400Response`? Because the error code examples is specific to a bad visitor ID?